getting_started/scripting/c_sharp/c_sharp_basics.rst

```diff
@@ -47,6 +47,22 @@ you can download just the
 instead.
 Make sure you at least have the .NET Framework 4.5 targeting pack installed, you can get it using any of the installers mentioned above inside the "Individual components" tab.
 
+Windows (JetBrains Rider)
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+JetBrains Rider comes with bundled MSBuild, so you just need recent .NET Framework Developer Pack.
```

Comment:
Would add an article here: *... so you just **a** need recent .NET Framework Developer Pack.*

Reply:
added it, thanks